test-nextjs/e2e-tests/README.md

```diff
@@ -1,18 +1,32 @@
-# Get the e2e tests working
+# Run the e2e tests
+
+## The mostly automated way
+
+The following instructions require some pre-work, but once you've done the first couple steps once
+you can skip to running the test command at the end. 
+
+1. Clone [convex-backend](https://github.com/get-convex/convex-backend)
+
+1. Follow the instructions in its [README](https://github.com/get-convex/convex-backend/blob/main/README.md) to get it building
+
+1. From the `test-nextjs` directory, run:
+
+```
+CONVEX_LOCAL_BACKEND_PATH=/path/to/your/convex-backend npm run test
+```
```

Comment:
Or use the built image? (easier)

Reply:
Total off-topic: I don't know if Prettier does it or what, but having the markdown lists be 1. (which works when rendered) makes it hard to read the source (which most devs will do).

Reply:
Thanks for the comments.

I'm going to leave the instructions as-is for now as they work and it's a one-time setup cost for someone who wants to run the end-to-end tests.

As for the Markdown list format, I just went with the convention that was already in the README.